### PR TITLE
Connect react and add first example component

### PR DESCRIPTION
**Pull request by solopiy**

### CODE REVIEW

**Comment by solopiy** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;Z9s5MeBC&#x2F;313-when-i-go-to-admin-panel-i-want-to-see-admin-panel-text-generated-by-react-so-we-can-develop-it-further"> When I go to &#x2F;admin-panel I want to see &quot;Admin panel&quot; text generated by React so we can develop it further</a>

File: app/assets/javascripts/admin.entry.js [hunk 1]
```diff
@@ -0,0 +1 @@
+import './containers/Example.jsx';
```
Comment (ezhlobo) [inline app/assets/javascripts/admin.entry.js]: I don't think that keeping containers in that path is good. Need something specific for admin. What do you think about `app/concepts/admin/ExampleContainer/ExampleContainer.jsx`?

File: app/assets/javascripts/containers/Example.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+class Example extends React.Component {
+    render(){
+        return (<h1>I am React</h1>);
```
Comment (ezhlobo) [inline app/assets/javascripts/containers/Example.jsx]: Unfortunately eslint still doesn't check parenthesis for tags in single line. But take a look at styleguide: https://github.com/airbnb/javascript/tree/master/react#parentheses

It suggests not to use parenthesis in a line.

File: package.json [hunk 3]
```diff
@@ -20,6 +21,11 @@
     "jquery-ujs": "^1.2.2",
     "node-sass": "^3.10.0",
     "postcss-loader": "^0.13.0",
+    "react": "^15.3.1",
+    "react-dom": "^15.3.1",
+    "react-redux": "^4.4.5",
+    "react-router": "^2.8.1",
+    "redux": "^3.6.0",
```
Comment (ezhlobo) [inline package.json]: Let's not to add dependencies until we use them

File: package.json [hunk 4]
```diff
@@ -45,6 +51,8 @@
     "eslint-plugin-import": "^1.15.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.2.2",
+    "react-router-redux": "^4.0.5",
+    "redux-devtools": "^3.3.1",
```
Comment (ezhlobo) [inline package.json]: I can't find where we added devtools. If we don't use it let's not to add it until we start using

File: app/assets/javascripts/admin.entry.js (hunk 1 above)
Comment (Proghat) [inline app/assets/javascripts/admin.entry.js]: @ezhlobo, what do you think, how will we organize the rest of files?

File: package.json [hunk 5]
```diff
@@ -45,6 +51,8 @@
     "eslint-plugin-import": "^1.15.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.2.2",
+    "react-router-redux": "^4.0.5",
```
Comment (Proghat) [inline package.json]: Why is `react-router-redux` placed in `devDependencies`?

File: app/assets/javascripts/admin.entry.js (hunk 1 above)
Comment (ezhlobo) [inline app/assets/javascripts/admin.entry.js]: @Proghat I'm not sure but I want to keep reducers, store and actions in `app/concepts/admin/(reducers|stores|actions|constants)`.

Comment (Proghat) [inline app/assets/javascripts/admin.entry.js]: Probably we will need to scope reducers, actions, etc per "component" or somehow. Take a look at components vs. modules in https://datarockets.com/blog/redux-structure.

Comment (ezhlobo) [inline app/assets/javascripts/admin.entry.js]: @Proghat yes it's ok. I said about root reducers, stores and actions. But we definitely should put all component's dependencies near the component as you said.

Comment (Proghat) [inline app/assets/javascripts/admin.entry.js]: @ezhlobo, what do you think, won't we have any problems with name collisions? There are a lot of subfolders in `app/concepts` already.

Comment (ezhlobo) [inline app/assets/javascripts/admin.entry.js]: @Proghat if we create `app/concepts/admin` and put there all files related to admin I think we will not have any problems

**Comment by Proghat** [comment]
What about general components like `Button`?

File: app/assets/javascripts/admin.entry.js (hunk 1 above)
Comment (Proghat) [inline app/assets/javascripts/admin.entry.js]: What about general components like `Button`?

Comment (ezhlobo) [inline app/assets/javascripts/admin.entry.js]: @Proghat now we won't use react components in the main website. So I think when we will store such components like `Button` in `app/concepts/admin/components/` we will not have any conflicts with naming. Please share your fears with me...

Comment (Proghat) [inline app/assets/javascripts/admin.entry.js]: @ezhlobo, I think you're right, let's do that 👍 .

File: config/assets_builder.js [hunk 6]
```diff
@@ -59,11 +62,20 @@ module.exports = {
...
                 exclude: /node_modules/,
             },
+            {
+                test: /.jsx$/,
+                loader: 'babel-loader',
+                exclude: /node_modules/,
+                query: {
+                    presets: ['react', 'es2015'],
+                },
+                include: path.resolve(__dirname, '../app/'),
```
Comment (ezhlobo) [inline config/assets_builder.js]: I see that this module is the same as above. You can just change value of `test` with `/\.(js|jsx)$/`

**Pull request merged**